it/config/ApiDocumentationControllerISpec.scala

```diff
@@ -82,7 +82,7 @@ class ApiDocumentationControllerISpec extends IntegrationBaseSpec {
            |        "endpointsEnabled": true,
            |        "access" : {
            |          "type": "PRIVATE",
-           |          "whitelistedApplicationIds": []
+           |          "allowlistedApplicationIds": []
```

Comment:
I removed it fromapplication.conf because I added default value when retrieving property in config class, so locally it will default always to empty there is nothing and when there is something like in tests then return that

Reply:
great - thanks